specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-07-19/quota.json

```diff
@@ -0,0 +1,1045 @@
+				"operationId": "QuotaRequests_GetStatus",
+				"parameters": [
+					{
+						"$ref": "#/parameters/SubscriptionIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/ApiVersionParameter"
+					},
+					{
+						"$ref": "#/parameters/ProviderIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/LocationIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/RequestIdInParameters"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "Ok. Returning the status for the quota request .",
+						"schema": {
+							"$ref": "#/definitions/QuotaRequestDetails"
+						}
+					},
+					"default": {
+						"description": "Error response describing why the operation failed.",
+						"schema": {
+							"$ref": "#/definitions/ExceptionResponse"
+						}
+					}
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Capacity/providers/{providerId}/locations/{location}/serviceLimitsRequests": {
+			"get": {
+				"tags": [
+					"QuotaRequests"
+				],
+				"x-ms-examples": {
+					"QuotaRequestHistory": {
+						"$ref": "./examples/getQuotaRequestsHistory.json"
+					}
+				},
+				"summary": "For the specified location and resource Provider, gets the quota requests under the subscription over the time\n period of one year ago from now to one year back, based on the filter specified.",
+				"description": "For the specified location and Resource provider gets the current quota requests under the subscription over the time period of one year ago from now to one year back. oData filter can be used to select quota requests.",
+				"operationId": "QuotaRequests_ListStatus",
+				"parameters": [
+					{
+						"$ref": "#/parameters/SubscriptionIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/ApiVersionParameter"
+					},
+					{
+						"$ref": "#/parameters/ProviderIdInParameters"
+					},
+					{
+						"$ref": "#/parameters/LocationIdInParameters"
+					},
+					{
+						"name": "$filter",
+						"in": "query",
+						"required": false,
+						"type": "string",
+						"description": "| Field                    | Supported operators  \n|---------------------|------------------------\n\r\n|requestSubmitTime | ge, le, eq, gt, lt\n"
+					},
+					{
+						"$ref": "#/parameters/TopQueryParameter"
+					},
+					{
+						"$ref": "#/parameters/SkipTokenParameter"
+					}
+				],
+				"x-ms-pageable": {
+					"nextLinkName": "nextLink"
+				},
+				"responses": {
+					"200": {
+						"description": "Ok. Sending Quota request status and details",
+						"schema": {
+							"$ref": "#/definitions/QuotaRequestDetailsList"
+						}
+					},
+					"default": {
+						"description": "Error response describing why the operation failed.",
+						"schema": {
+							"$ref": "#/definitions/ExceptionResponse"
+						}
+					}
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Capacity/autoQuotaIncrease": {
```

Comment:
what is autoQuotaIncreate?

Reply:
AutoQuotaIncrease is a feature for a subscription, where user can opt in to automatically increase the quota, as needed.